README.md

```diff
@@ -13,8 +13,48 @@ This textbook gives scientists:
 - a guide to designing parallel processing algorithms to work efficiently with chunked datasets
 - a guide to exporting chunked datasets to other 'tradditional' datasets
 
+## Benchmarking for Zarr
+
+We created a set of benchmarks for writing data to Zarr with a range of different configurations as guidance for the options available when reading and writing 3D imaging data.
+The different parameters were:
+
+- Type of image
```

Comment:
Here you're mixing the different parameters (e.g. type of image = heart / dense /sparse) with benchmarking results (e.g. compressor = blosc-zstd provides the best compression ratio). Maybe just list the different parameters options here, and link to the final report for results? Not sure what @dstansby 's preference is.